framework/Theme.php

```diff
@@ -113,6 +113,13 @@ abstract class Theme {
 		'caption',
 	);
 
+	/**
+	 * Disable gutenberg for posts and custom post type.
+	 *
+	 * @var array $disable_gutenberg_post_type
+	 */
+	public $disable_gutenberg_post_type = [];
```

Comment:
I prefer to keep the name simple - disable_gutenberg. 
It can be boolean, or array (need specific comment).

In case boolean - you can disable it totally (so you need to disable for all post types).
In case array - you disable only specific onces